src/views/consent/ConsentButtonsBar.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta'
+], function (Okta) {
+
+  return Okta.View.extend({
```

Comment:
do we really need this file? It is not doing anything special with 'save' or 'cancel'. Can we get away with just passing the strings we need for those two buttons in the ```ConsentRequiredController``` views?

Reply:
will remove this for the beta when I will put the real buttons